src/current/v23.1/migration-overview.md

```diff
@@ -5,87 +5,304 @@ toc: true
+
+- *Planned downtime* is announced to your users in advance. Once you have [prepared for the migration](#prepare-for-migration), you take the application offline, [conduct the migration](#conduct-the-migration), and bring the application back online on CockroachDB. To succeed, you should estimate the amount of downtime required to migrate your data, and ideally schedule the downtime outside of peak hours. Scheduling downtime is easiest if your application traffic is "seasonal", meaning that it is less active during predictable hours.
+
+- *Unannounced downtime* impacts as few customers as possible, ideally without impacting their regular usage. If your application is intentionally offline at certain times (e.g., outside business hours), you can migrate the data without users noticing. Alternatively, if your application's functionality is not time-sensitive (e.g., it sends batched messages or emails), then you can queue requests while your system is offline, and process those requests after completing the migration to CockroachDB.
+
+- *Reduced functionality* takes some, but not all, application functionality offline. For example, you can disable writes but not reads while you migrate the application data, and queue data to be written after completing the migration.
+
+For an overview of lift-and-shift migrations to CockroachDB, see [Lift and Shift](#lift-and-shift). {% comment %}For details, see [Migration Strategy: Lift and Shift](migration-strategy-lift-and-shift).{% endcomment %}
+
+#### Zero or near-zero downtime
+
+If your application cannot tolerate downtime, then you should aim for a "zero-downtime" approach. "Zero" means that downtime is reduced to either an absolute minimum or zero, such that users do not notice the migration.
+
+The minimum possible downtime depends on whether you can tolerate inconsistency in the migrated data:
+
+- *Consistent* migrations reduce downtime to an absolute minimum (i.e., from 30 seconds to sub-seconds) while keeping data synchronized between the legacy database and CockroachDB. **Consistency requires downtime.** In this approach, downtime occurs right before [cutover](#cutover-strategy), as you drain the remaining transactions on the legacy database.
+
+- *Inconsistent* migrations reduce downtime to zero. These require the most preparation, and typically involve "dual writing" to both databases in lockstep. This means forking the workload and applying writes to both the legacy database and CockroachDB, side-by-side. {% comment %}You can use the CockroachDB Live Migration Service (MOLT LMS) to run application queries simultaneously on your legacy database and CockroachDB.{% endcomment %} Without stopping application traffic, you perform an immediate [cutover](#cutover-strategy), while assuming that some writes will not be replicated to CockroachDB. You must manually reconcile these data inconsistencies after switching over.
```

Comment:
I think we need to say the LMS is in private preview here but I like that we are including it from the beginning. We can say contact sales to be a part of the private preview.

Reply:
Actually the `{% comment %}` tags are intended to hide this on the page until we're ready to reveal - see the preview link: https://deploy-preview-17339--cockroachdb-docs.netlify.app/docs/v23.1/migration-overview.html

I'm not sure we have a docs precedent or policy for mentioning features that are in private preview / limited access. @lnhsingh @mikeCRL can you comment on whether Andy's suggestion sounds ok for our public docs?

Reply:
Better to wait on any public mention of LMS until it goes into Limited Access per: